Examples/Python/python/acts/examples/itk.py

```diff
@@ -445,9 +445,7 @@ def itkSeedingAlgConfig(
         if highOccupancyConfig == True:
             rMaxGridConfig = 250 * u.mm
             deltaRMax = 200 * u.mm
-            zBinsCustomLooping = [1, 11, 2, 10, 3, 9, 6, 4, 8, 5, 7]
-            # variables that are only used for highOccupancyConfig configuration:
-            skipZMiddleBinSearch = 2
+            zBinsCustomLooping = [2, 10, 3, 9, 6, 4, 8, 5, 7]
```

Comment:
Sorry, I'm slow. What does this end up doing? Are these global middle z bins that the iterator visits? Does this not depend on the seed combination?

Reply:
This basically allows you to search for the middle space point only in some regions of the detector, as defined by the grid. Also, you can now specify these custom loopings for any axis in the grid ... potentially you can do seeding only in the forward region of the detector by being smart with these arrays. If you add an r-axis to the grid you can also only loop in the innermost part of the detector with no `c++` change